core/ui/public/i18n/en/translation.json

```diff
@@ -575,6 +584,31 @@
     "settings_cluster": {
         "title": "Cluster"
     },
+    "settings_subscription": {
+        "title": "Subscription",
+        "cluster_subscription": "Cluster subscription",
+        "cluster_subscription_description": "The cluster subscription is used to enable the cluster features and to receive updates for the cluster nodes.",
```

Comment:
Maybe we should clarify this sentence, we are not saying which cluster features will be enabled. My proposal: "Cluster subscription activates extra cluster features, such as automatic updates for the cluster core and the apps"